packages/util/test/base64.test.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { assert } from 'chai';
+import { base64Encode, base64Decode } from '../src/crypt';
+
+describe('base64', () => {
+  it('bijective', () => {
+    let str = '…';
+    assert.strictEqual(base64Decode(base64Encode(str)), str);
```

Comment:
Can we test each of the variants? So a 2-byte UTF-8 char, a 3-byte char and a 4 byte char? (ascii too).

Reply:
Added cases for 1--4 bytes